fs/fs.go

```diff
@@ -45,15 +45,24 @@ const (
 )
 
 // The maximum number of `du` tasks that can be running at once.
-const maxConsecutiveDus = 20
+const maxConcurrentDus = 20
+
+// The maximum number of `find` tasks that can be running at once.
+const maxConcurrentFinds = 20
```

Comment:
Do we need a separate pool for each of these? On the one hand, if one operation is significantly slower, it prevents it from hogging the tokens. On the other hand, the operations happen in sequence, so the one operation will be blocking anyway. I'm leaning towards a single "exec" pool.